### PR TITLE
Esutils cleanup

### PR DESCRIPTION
**Pull request by aalsabag**
This PR:
1. Adds `created` and `updated` timestamp fields for policy
2. Sort list policies by create time
3. Sort resources alphabetically
4. Add a default client only to opa with a timeout of five seconds
5. Uses only esutil types. They have been ported over to grafeas-elasticsearch

### CODE REVIEW

File: opa/opa.go [hunk 1]
```diff
@@ -37,6 +38,7 @@ type client struct {
 	logger       *zap.Logger
 	Host         string
 	ExplainQuery bool
+	HTTPClient   *http.Client
```
Comment (alexashley) [inline opa/opa.go:41]: Nitpick, but the client should probably be a private field.

File: server/server.go [hunk 2]
```diff
@@ -307,6 +307,7 @@ func (r *rodeServer) ListResources(ctx context.Context, request *pb.ListResource
 		r.esClient.Search.WithIndex(rodeElasticsearchOccurrencesAlias),
 		r.esClient.Search.WithBody(encodedBody),
 		r.esClient.Search.WithSize(maxPageSize),
+		r.esClient.Search.WithSort("resource.uri:asc"),
```
Comment (alexashley) [inline server/server.go:310]: Could/should this use the [sort](https://github.com/rode/grafeas-elasticsearch/blob/fbd2243fbf82e6c40ff5ff35297b75a3fcfa3194/go/v1beta1/storage/esutil/types.go#L51) field on the `EsSearch` struct now?

**Review by alexashley — COMMENTED** [review]
LGTM, just some really minor comments.

File: server/server.go (hunk 2 above)
Comment (aalsabag) [reply server/server.go:310]: It sure can!

File: opa/opa.go (hunk 1 above)
Comment (aalsabag) [reply opa/opa.go:41]: will do!

**Review by alexashley — APPROVED** [review]
:shipit:

**Pull request merged**